src/server/game/AuctionHouse/AuctionHouseMgr.cpp

```diff
@@ -648,105 +648,128 @@ void AuctionHouseObject::BuildListOwnerItems(WorldPacket& data, Player* player,
 void AuctionHouseObject::BuildListAuctionItems(WorldPacket& data, Player* player,
     std::wstring const& wsearchedname, uint32 listfrom, uint8 levelmin, uint8 levelmax, uint8 usable,
     uint32 inventoryType, uint32 itemClass, uint32 itemSubClass, uint32 quality,
-    uint32& count, uint32& totalcount)
+    uint32& count, uint32& totalcount, bool getall)
 {
     int loc_idx = player->GetSession()->GetSessionDbLocaleIndex();
     int locdbc_idx = player->GetSession()->GetSessionDbcLocale();
 
     time_t curTime = sWorld->GetGameTime();
+    time_t throttleTime = GetAllThrottleMap.find(player->GetGUID()) != GetAllThrottleMap.end() ? GetAllThrottleMap[player->GetGUID()] : curTime;
```

Comment:
I did consider that. But, wasn't sure how well adding to player would be, for a minor little used AH feature.

All the information to support this has been around for a while, I suspect it wasn't added because most people didn't really know about it. I doubt it would get a lot of use. Hence I thought a separate map might be a better solution.

But, maybe a straw poll to decide which way is better and I'll then make the change if required.